easybuild/tools/toolchain/compiler.py

```diff
@@ -27,6 +27,7 @@
 
 :author: Stijn De Weirdt (Ghent University)
 :author: Kenneth Hoste (Ghent University)
+:author: Damian Alvarez (Forschungszentrum Juelich GmbH)
```

Comment:
There's no strict policy, but if you made beyond trivial changes to a file it makes sense to include yourself as an author; it gives people a good idea on who to talk to in case of questions.